src/aspire/nufft/finufft.py

```diff
@@ -5,7 +5,7 @@
 
 from aspire.nufft import Plan
 from aspire.utils import ensure
-from aspire.utils.misc import complex_type
+from aspire.utils.types import complex_type, dtype_legacy_string
```

Comment:
`dtype_legacy_string` is unused.

Reply:
Oops, leftover from rebasing this (originally finufftv2 was implemented before this change set here, but..). Slipped through. Will rm.